src/foam/i18n/ClientCacheTranslationService.js

```diff
@@ -116,6 +125,13 @@ foam.CLASS({
         this.translation.pub(locale, source, txt, defaultText);
         return txt || defaultText;
       }
+    },
+    function loadTheme() {
+      return this.localeDAO.where(
+        this.AND(
```

Comment:
Needs to load locales with variant specified as well as an empty variant since we would need fallbacks